api/v1alpha3/conversion.go

```diff
@@ -104,14 +141,35 @@ func (r *AWSMachineList) ConvertFrom(srcRaw conversion.Hub) error {
 func (r *AWSMachineTemplate) ConvertTo(dstRaw conversion.Hub) error {
 	dst := dstRaw.(*v1alpha4.AWSMachineTemplate)
 
-	return Convert_v1alpha3_AWSMachineTemplate_To_v1alpha4_AWSMachineTemplate(r, dst, nil)
+	if err:= Convert_v1alpha3_AWSMachineTemplate_To_v1alpha4_AWSMachineTemplate(r, dst, nil); err !=nil{
+		return err
+	}
+
+	restored := &v1alpha4.AWSMachineTemplate{}
+	if ok, err := utilconversion.UnmarshalData(r, restored); err != nil || !ok {
+		return err
+	}
+
+	if restored.Spec.Template.Spec.RootVolume != nil && restored.Spec.Template.Spec.RootVolume.Encrypted == nil {
+		dst.Spec.Template.Spec.RootVolume.Encrypted = nil
+	}
+
+	for i := range restored.Spec.Template.Spec.NonRootVolumes {
+		if restored.Spec.Template.Spec.NonRootVolumes[i].Encrypted == nil {
+			dst.Spec.Template.Spec.NonRootVolumes[i].Encrypted = nil
+		}
+	}
```

Comment:
Instead of copy pasting the same code as for AWSMachine, can create a 

`Convert_v1alpha3_AWSMachineSpec_To_v1alpha4_AWSMachineSpec` and reuse the same code.